client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -1122,47 +1109,21 @@ public void addFailuresShuffleServerInfos(String shuffleServerId) {
   public synchronized boolean reassignShuffleServers(
```

Comment:
Can you rename this method to `reassignAllShuffleServersForWholeStage` ,current name is similar with the `reassignFaultyShuffleServer`, that is not clear

Reply:
It's ok for me.